app/forms/idv/api_image_upload_form.rb

```diff
@@ -309,5 +294,26 @@ def throttle
         throttle_type: :idv_doc_auth,
       )
     end
+
+    def track_event(response)
+      pii_from_doc        = response.pii_from_doc || {}
+      stored_image_result = store_encrypted_images_if_required
```

Comment:
we mostly do not do the "lining up =" style in this app, it can be really noisy over time
```suggestion
      pii_from_doc = response.pii_from_doc || {}
      stored_image_result = store_encrypted_images_if_required
```